src/pywws/service/mqtt.py

```diff
@@ -92,9 +116,17 @@
 ``/weather/pywws/temp_in_c``.
 
 ``template_txt`` is the template used to generate the data to be
-published. You can edit it to suit your own requirements, e.g. not using
-antiquated units of measurement. Be very careful about the backslash
-escaped quotation marks though.
+published. You can edit it to suit your own requirements. Be very careful
+about the backslash escaped quotation marks though. If not specified
+default will be used, which sends a lot of values in metric and imperial
+units.
+
+.. versionchanged:: NN.N
```

Comment:
Please let me know what version should  I put here or if you are happy to update it once merged?

Reply:
I'll set it when I do the next release. That's when I normally modify the changelog, using GitHub's "changes since the last release" to remind me what to say.